client/app/scripts/liveblog-edit/module.js

```diff
@@ -173,6 +172,12 @@ define([
                     return true;
                 }
             }
+
+        });
+        blogSecurityService.isUserOwner().then(function(result) {
+            $scope.blog.showSettingsLink = result;
+        }, function() {
+            $scope.blog.showSettingsLink = false;
```

Comment:
To avoid this kind of instructions lost in the controller, I would set it into the template directly where you need it.
```html
ng-if="blogSecurityService.isUserOwner(blog)"
```

As you can see, I would also add a `blog` parameter, because I think this function should be usable outside from the route context.